packs/feat-effects/effect-heavens-thunder.json

```diff
@@ -50,6 +50,58 @@
                 "selector": "strike-damage",
                 "slug": "heavens-thunder-sonic",
                 "value": "@weapon.system.damage.dice"
+            },
+            {
+                "alwaysActive": true,
```

Comment:
Since you can choose to deal electricity or sonic damage, this shouldn't be always active.